tasks/install.yml

```diff
@@ -1,58 +1,50 @@
 ---
-- name: Check if pi-hole is already installed
-  ansible.builtin.stat:
-    path: "/opt/pihole"
-  register: pihole_installed
-  tags:
-    - pihole
-    - installation
-    - check
-
 - name: Create pihole config directory
   become: true
   ansible.builtin.file:
     name: /etc/pihole
     state: directory
-    mode: 0644
-  when: pihole_installed.stat.isdir is undefined
+    mode: "0664"
   tags:
     - pihole
     - installation
     - prepare
 
 - name: Copy pihole setupVars
   become: true
+  register: __pihole_setupVars
```

Comment:
if `pi_hole_webpasswd` is not defined for the host, `pi-hole` will create one and add it to `setupVars.conf`
This make test not idempotent.
I don't remember that it was failing before :thinking:
Maybe something has changed on `pi-hole` since i worked on it

Reply:
Good point. I didn't test it with no password, as I never rely on generated passwords during the setup process.

Do you have any suggestions to solve this?

Reply:
Maybe make webpassword required

Reply:
> Maybe make webpassword required

I would prefer this. What's you opinion about this (breaking) change?

Reply:
I'm pretty quite sure post of people define their own password. 
So we just need to release a mohair version